src/main/java/net/datafaker/GarmentSize.java

```diff
@@ -0,0 +1,76 @@
+package net.datafaker;
+
+/**
+ * This class is used to generate garments sizes randomly.
+ *
+ */
+
+public class GarmentSize extends AbstractProvider {
+
+  protected GarmentSize(Faker faker) {
+    super(faker);
+  }
+
+  /**
+   * This method returns a garment size
+   *
+   * @return a string of garment size
+   */
+  public String size() {
+    return faker.fakeValuesService().fetchString("garments.sizes");
+  }
+
+  /**
+   * This method returns a garment XS size
+   *
+   * @return a XS string of garment size
+   */
+  public String anXS() { return faker.fakeValuesService().fetchString("garments.xs"); }
```

Comment:
I don't really understand this. Why would you want this?

Reply:
I just want a `size` method to get a random size, and a specific method for each size to get a specific size.

Reply:
But if you know which size you want, why do even want to call the method?

Reply:
totally true... make no sense... sorry abut that.. i did a commit to solved this.